app/authentication_views.py

```diff
@@ -90,7 +92,7 @@ def login():
                 f"Handling '/login' route, authenticating user {username} failed: Invalid password")
             return "invalid password"
 
-        session["user"] = user.to_json()
+        logginManager.login_user(user)
```

Comment:
log_in_user
login is like username
and maybe rename to authenticate?

while managing session you have 2 big concepts
authentication and authorization
authentication is to know that the person that wants to log in is user with id x stored in db
authorization is checking if that user can access given resource.

here it looks like you are doing only authentication so maybe it is worth to call that method authenticate

Reply:
done